flask_app.py

```diff
@@ -140,7 +140,7 @@ def get_recordby_column_name():
     return data
 
 
-@app.route('/update_record_by_column', methods=['GET','POST'])
+@app.route('/update_record_by_column', methods=['PUT'])
 @check_authorization
 def get_update_by_column_name():
     table_name = request.args.get('table_name')
```

Comment:
Null checking or type checking?

Reply:
Added in the exception handling code reflected on this changeset

Reply:
e1ae681